lib/cerner/oauth1a/access_token_agent.rb

```diff
@@ -19,14 +19,25 @@ module OAuth1a
     class AccessTokenAgent
       MIME_WWW_FORM_URL_ENCODED = 'application/x-www-form-urlencoded'
 
+      DEFAULT_REALM_ALIASES = {
```

Comment:
I can't find documentation for the EU OAuth server. 

All of the services I can see configuration for use `oauth-api.eu.cerner.com`. However, I do see a couple repos that look more like support/troubleshooting scripts that have `api.eu.cernercare.com`; not sure if that host actually works or not.

Reply:
I can't find any evidence that 'api.eu.cernercare.com' really exists or ever did. We can add more aliases later, if needed.